dev_tools/auto_merge.py

```diff
@@ -529,6 +536,14 @@ def auto_delete_pr_branch(pr: PullRequestDetails) -> bool:
         return False
 
     remote = pr.remote_repo
+    if (pr.repo.organization.lower() != remote.organization.lower() or
+            pr.repo.name.lower() != remote.name.lower()):
+        print('Not deleting branch {!r}. It belongs to a fork ({}/{}).'.format(
```

Comment:
so if the repo name is different but the organization is the same it won't delete it?  so branches created on quantumlib aren't deleted?

Reply:
Correct. If you somehow forked into a different repo within quantumlib and then merged in, this would not delete that branch. I think that's the correct behavior.